packages/asc-ui/src/components/TopTaskButton/TopTaskButton.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import TopTaskButtonStyle, { Props } from './TopTaskButtonStyle'
+import Icon from '../Icon'
+
+const TopTaskButton: React.FC<Props> = ({
+  title,
+  content,
+  icon,
+  ...otherProps
+}) => {
+  const CustomIcon = icon
+
+  return (
+    <TopTaskButtonStyle {...otherProps}>
+      <Icon inline size={32} color="black">
+        <CustomIcon />
+      </Icon>
+      <div>{title}</div>
+      <section>{content}</section>
```

Comment:
Why the section? Can we use `<Paragraph>` here?

Reply:
done